packages/api/src/templates/locales/fr.json

```diff
@@ -97,5 +101,6 @@
     "SUBJECT": "EXPORT.SUBJET",
     "PRIVACY": "EXPORT.CONFIDENTIALITÉ",
     "LANGUAGE_DELAY": "EXPORT.DÉLAIS.LANGUE"
```

Comment:
These three look like they were not translated properly in the past, can you mark them as "MISSING"  and I'll add them later when I do another pass of FR translations?